filebeat/crawler/registrar.go

```diff
@@ -212,28 +199,15 @@ func (r *Registrar) getPreviousFile(newFilePath string, newFileInfo os.FileInfo)
 }
 
 func (r *Registrar) setState(path string, state *FileState) {
```

Comment:
I worry I will potentially need them again in the near future. I would like to keep them in until the refactoring is completed. In the end it should be possible to remove them.